react/src/AddDatasets/index.tsx

```diff
@@ -28,21 +28,33 @@ const useStyles = makeStyles(theme => ({
     },
 }));
 
-export default function AddParticipants() {
+export default function AddParticipants(props: {
+    groups: string[]; // groups that the user is in
+}) {
     const classes = useStyles();
     const history = useHistory();
     const [data, setData] = useState<DataEntryRow[]>([]);
     const [open, setOpen] = useState(false);
     const [errorMessage, setErrorMessage] = useState("");
+    const [asGroups, setAsGroups] = useState<string[]>([]); // "submitting as these groups"
     const { enqueueSnackbar } = useSnackbar();
 
     useEffect(() => {
         document.title = `Add Datasets | ${process.env.REACT_APP_NAME}`;
         handleDataChange(createEmptyRows(1)); // sets errorMessage on initial render
     }, []);
 
+    useEffect(() => {
+        if (props.groups.length > 0) {
+            setAsGroups(props.groups);
+        } else {
+            setErrorMessage("Cannot submit. You are not part of any permission groups.");
```

Comment:
Can a similar message be added for when the user attempts to submit with the field not filled out?